packages/babel-preset-typescript/README.md

```diff
@@ -0,0 +1,54 @@
+# babel-preset-typescript
+
+> Babel preset for TypeScript.
+
+This preset includes the following plugins:
+
+- [transform-typescript](https://babeljs.io/docs/plugins/transform-typescript/)
+- [syntax-object-rest-spread](https://babeljs.io/docs/plugins/syntax-object-rest-spread/)
```

Comment:
We just want to enable syntax, not add any transforms. That way people can have an emit that includes class properties (minus types) if they want to.